bigquery/start.py

```diff
@@ -0,0 +1,224 @@
+import os.path
+import requests, json
+
+from google.auth.transport.requests import Request
+from google.oauth2.credentials import Credentials
+from google_auth_oauthlib.flow import InstalledAppFlow
+from googleapiclient.discovery import build
+from googleapiclient.errors import HttpError
+
+# If modifying these scopes, delete the file token.json.
+SCOPES = ["https://www.googleapis.com/auth/bigquery"]
+
+WEATHER_API_KEY = "YOUR_KEY"
```

Comment:
Shouldn't this go in the weather cloud function?

Reply:
It is just a temporary way to collect weather day. The API needs latitude and longitude, so I save the mapping of city names to coordinates into the database initially. However, after we define segments, I believe we do not need this because we can get the coordinates from segment definitions. Therefore, I will just delete this